xds/third_party/envoy/import.sh

```diff
@@ -18,7 +18,7 @@
 set -e
 BRANCH=main
 # import VERSION from one of the google internal CLs
-VERSION=ac9a2637336decdcc52c24add5e8fc39edebb962
+VERSION=8b9b87702885beb324dadb349cbcb06d037c956e
```

Comment:
Is bumping the version necessary for importing rbac protos?

Reply:
Good question. Looks like not necessary. The only difference i noticed in `v3.rbac` is just deprecation annotation changes. And then all the dependency chain. Shall we be careful about bumping up envoy version? Things added like `quic_config.proto` looks something that we never need.

Reply:
>Shall we be careful about bumping up envoy version?

If we only add protos, without bumping the version, the PR will look much simple. If our version is too out of date from google3, then bumping the version is fine.